packages/api/src/utils/createTask.ts

```diff
@@ -649,6 +651,12 @@ export const enqueueRssFeedFetch = async (
     folders: subscriptionGroup.folders,
   }
 
+  let jobid = `refresh-feed_${stringToHash(
+    subscriptionGroup.url
+  )}_${stringToHash(JSON.stringify(subscriptionGroup.userIds.sort()))}`
```

Comment:
Do we need user id here after we already group the subscriptions by url?

Reply:
@sywhb I think we still do, because when the feed is refreshed its added to the supplied list of users. So imagine we refreshed the feed by just URL, then another user added the feed, then we refreshed again. I think if the key was still "in flight" it would generate two entries, right?

Reply:
yeah, you are right